cypress/e2e/log-in.cy.ts

```diff
@@ -4,28 +4,29 @@ describe('User Authentication Tests', () => {
 
   // Check if the user exists and can successfully log in
   it('should log in an existing user and redirect to the user page', () => {
-    cy.visit('http://localhost:5173/log-in');
+    cy.visit('http://localhost:5173/UserLogin');
 
     // Type the email and password for an existing user(this is just example placeholder)
-    cy.get('[data-test="signin-email"]').type('existingUser@example.com');
-    cy.get('[data-test="signin-password"]').type('validPassword');
+    cy.get('[name="username"]').type('existingUser');
+    cy.get('[name="password"]').type('validPassword');
 
 
-    cy.get('[data-test="signin-submit"]').click();
+    cy.get('button[type="submit"]').click();
 
     cy.location("pathname").should("equal", "/UserPage");
   });
 
   // Handle unsuccessful login and display an error message
   it('should display an error message for invalid credentials', () => {
-    cy.visit('http://localhost:5173/log-in');
+    cy.visit('http://localhost:5173/UserLogin');
 
-    cy.get('[data-test="signin-email"]').type('invalidUser@example.com');
-    cy.get('[data-test="signin-password"]').type('wrongPassword');
+    cy.get('[name="username"]').type('existingUser');
+    cy.get('[name="password"]').type('validPassword');
 
-    cy.get('[data-test="signin-submit"]').click();
+    cy.get('button[type="submit"]').click();
 
-    cy.get('[data-test="signin-error"]').should('be.visible')
+    // error message after login atempt
+    cy.get('.error-message').should('be.visible')
```

Comment:
```suggestion
    cy.get('[class=.error-message]').should('be.visible')
```
could this fix the issue?